src/internal_events/postgresql_metrics.rs

```diff
@@ -4,6 +4,37 @@ use super::prelude::{error_stage, error_type};
 use metrics::{counter, histogram};
 use vector_core::internal_event::InternalEvent;
 
+#[derive(Debug)]
+pub struct PostgresqlMetricsEventsReceived<'a> {
+    pub byte_size: usize,
+    pub count: usize,
+    pub endpoint: &'a str,
+}
+
+impl<'a> InternalEvent for PostgresqlMetricsEventsReceived<'a> {
```

Comment:
Is this event actually used now? I can't find a reference in these sources.

Reply:
Weird, not sure how Clippy missed this both locally and in CI... but you're right, it's not used anymore. I'll remove it.

Reply:
It's not caught because all of these internal events are fully `pub` (too much pain in component feature checks).